packages/path_provider/path_provider_android/README.md

```diff
@@ -0,0 +1,29 @@
+# path_provider
```

Comment:
This `README` is still the exact same as the one in `path_provider`. Is the plan to fix this in the followup PR?

Reply:
Oops, that's just copy-pasta. Thanks for catching that!

Reply:
Fixed. It's now using exactly the same format as the existing implementation package READMEs.

(It says they are endorsed, which isn't technically true at the moment they are published, but will be immediately after since they aren't going to have a deliberately "available as unendorsed" period as we've sometimes done with plugins (e.g., on web), so having them say it now avoids pointless churn.)